I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/places.rpy", line 16, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_all_missions(1)
  File "game/base_of_operations.rpy", line 64, in script call
    call see_all_missions(1)
  File "game/base_of_operations.rpy", line 23, in script call
    call see_missions_for_region(1)
  File "game/base_of_operations.rpy", line 68, in script
    call screen mission_screen(regionNumber)
Exception: renpy.restart_interaction() was called 100 times without processing any input.

-- Full Traceback ------------------------------------------------------------

Traceback (most recent call last):
  File "game/places.rpy", line 16, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_all_missions(1)
  File "game/base_of_operations.rpy", line 64, in script call
    call see_all_missions(1)
  File "game/base_of_operations.rpy", line 23, in script call
    call see_missions_for_region(1)
  File "game/base_of_operations.rpy", line 68, in script
    call screen mission_screen(regionNumber)
  File "renpy/ast.py", line 2231, in execute
    self.call("execute")
    ~~~~~~~~~^^^^^^^^^^^
  File "renpy/ast.py", line 2195, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/statements.py", line 379, in call
    return method(parsed, *args, **kwargs)
           ~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/common/000statements.rpy", line 695, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
                    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/exports/statementexports.py", line 348, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
         ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/ui.py", line 304, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/display/core.py", line 2217, in interact
    raise Exception("renpy.restart_interaction() was called 100 times without processing any input.")
Exception: renpy.restart_interaction() was called 100 times without processing any input.

Windows-11-10.0.26120-SP0 AMD64
Ren'Py 8.4.1.25072401
Protectors 1.0
Tue Aug 19 16:10:20 2025
